utils/mail_service.py

```diff
@@ -0,0 +1,50 @@
+from urllib.parse import urlencode
+import os
+from fastapi_mail import ConnectionConfig, FastMail, MessageSchema, MessageType
+from starlette.responses import JSONResponse
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Define your email configuration
+conf = ConnectionConfig(
+    MAIL_USERNAME=os.getenv("USERNAME", "AnyName"),
+    MAIL_PASSWORD=os.getenv("MAIL_PASSWORD", "md-XnF_xUedhL_K8aQFqgnneQ"),
+    MAIL_FROM=os.getenv("MAIL_FROM", "neilohene@gmail.com"),
+    MAIL_PORT=os.getenv("MAIL_PORT", 587),
+    MAIL_SERVER=os.getenv("MAIL_SERVER", "smtp.mandrillapp.com"),
+    MAIL_STARTTLS=True,
+    MAIL_SSL_TLS=False,
+    USE_CREDENTIALS=True,
+    VALIDATE_CERTS=True
+)
+
+# Setup mail service with MailChimp SMTP & API Credentials
+
+async def send_email(email: str, reset_token: str) -> JSONResponse:
+    reset_password_url = f"http://127.0.0.1:8080/reset-password?{urlencode({'token': reset_token})}"
```

Comment:
I think we need to set a base URL somewhere in the env file and access it in the settings file before using here.

Reply:
Sure. I'll do that